include/ginkgo/core/solver/gmres.hpp

```diff
@@ -54,7 +54,7 @@ namespace gko {
 namespace solver {
 
 
-constexpr size_type default_krylov_dim = 100u;
+constexpr size_type gmres_default_krylov_dim = 100u;
```

Comment:
Maybe it's considered as public interface

Reply:
What do you mean by this comment? I updated the name only because there is another `default_krylov_dim` in GCR.

Reply:
`default_krylov_dim` is in the public interface (namespace gko::solver). Deleting or renaming it is public interface change, so we should avoid it. you can use `default_krylov_dim` for gmres and `gcr_default_krylov_dim` for gcr. Or, add `gmres_default_krylov_dim` and deprecated the `default_krylov_dim`.